sql/normandy_recipe_versions_v1.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  f.key AS recipe_id,
+  f.value AS recipe_revision_id,
+  COUNT(f.value) AS count
+FROM
+  `moz-fx-data-derived-datasets.telemetry.main_summary_v4` m,
```

Comment:
the dataset and project are implied, and making them explicit is breaking your test
```suggestion
  main_summary_v4 m,
```

Reply:
Are you referring to the 21 failures with `google.auth.exceptions.DefaultCredentialsError: Could not automatically determine credentials` ? I made the change but it doesn't seem to have fixed anything.

Reply:
tests can't run at all on PRs from forked repos, due to lack of credentials, that is what you are seeing.

if you fixed that, you would eventually get to an error due to lack of permission to see actual main summary, because you are fully qualifying it here.